lib/install/management/appliance.go

```diff
@@ -113,7 +113,12 @@ func (d *Dispatcher) checkExistence(conf *config.VirtualContainerHostConfigSpec,
 	defer trace.End(trace.Begin(""))
 
 	var err error
-	d.vchPoolPath = path.Join(settings.ResourcePoolPath, conf.Name)
+	// TODO: Remove when inventory folder support complete #7032
+	if conf.HostSelectionRequired {
```

Comment:
Suggest just switching this based on DRS enabled state - it will still need to be done when the folder support is in place as we still want to support resource pools for non-ROBO installs.

Reply:
Agreed